Cargo.toml

```diff
@@ -19,6 +19,7 @@ notify = { version = "5.0.0-pre.2", optional = true }
 math = { path = "../math" }
 #core = { git = "https://github.com/roadrunner-craft/core" }
 core = { path = "../core" }
+log = { version = "0.4.8", features = ["std"] }
```

Comment:
use re-export from the core instead of depending on the log crate here

Reply:
good call, did that